crates/ty_python_semantic/src/types/class.rs

```diff
@@ -2824,7 +2831,7 @@ impl<'db> ClassLiteral<'db> {
     /// Returns a list of all annotated attributes defined in this class, or any of its superclasses.
     ///
     /// See [`ClassLiteral::own_fields`] for more details.
-    #[salsa::tracked(returns(ref), heap_size=get_size2::GetSize::get_heap_size)]
+    #[salsa::tracked(returns(ref), cycle_initial=fields_cycle_initial, heap_size=get_size2::GetSize::get_heap_size)]
```

Comment:
nit: quite a long line

```suggestion
    #[salsa::tracked(
        returns(ref),
        cycle_initial=fields_cycle_initial,
        heap_size=get_size2::GetSize::get_heap_size)
    ]
```